docs/docs-content/clusters/pcg/deploy-pcg/vmware.md

```diff
@@ -0,0 +1,550 @@
+    | **Spectro Cloud Organization** | Select your Palette Organization name.                                                                                                                                                                                                                            |
+    | **Spectro Cloud Project**      | Select the project name you want to register the VMware vSphere account in.                                                                                                                                                                                       |
+    | **Acknowledge**                | Accept the login banner message. Login banner messages are only displayed if the tenant admin enabled a login banner.                                                                                                                                             |
+
+    :::info
+
+    The `CloudAccount.apiKey` and `Mgmt.apiKey` values in the **pcg.yaml** are encrypted and cannot be manually updated.
+    To change these values, use the `palette pcg install --update-passwords` command. Refer to the
+    [PCG command](../../../palette-cli/commands/pcg.md#update-passwords) reference page for more information.
+
+    :::
+
+3.  Once you have authenticated successfully, start the PCG installer by issuing the following command. Refer to the
+    table below for information about each parameter.
+
+    ```bash
+    palette pcg install
+    ```
+
+    | **Parameter**                                        | **Description**                                                                                                                                                                                                                                  |
+    | :--------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+    | **Management Plane Type**                            | Select Palette or VerteX.                                                                                                                                                                                                                        |
+    | **Enable Ubuntu Pro (required for production)**      | Choose `y` if you want to to use Ubuntu Pro, otherwise press `n`. If you select `y`, you will be be asked to provide an Ubuntu Pro token.                                                                                                        |
```

Comment:
```suggestion
    | **Enable Ubuntu Pro (required for production)**      | Enter `y` if you want to to use Ubuntu Pro and provide an Ubuntu Pro token. Otherwise, enter `n`.                                                                                                        |
```